climada/entity/exposures/crop_production.py

```diff
@@ -481,180 +490,201 @@ def init_full_exposure_set(input_dir=INPUT_DIR, filename=None, hist_mean_dir=HIS
-        cropyield.write_hdf5(os.path.join(output_dir, 'Exposure', filename_saveto))
-
-    if returns == 'filename_list':
+    for file in filenames:
+        histmean_prop = file.split('_')
+        crop_production = CropProduction()
+        crop_production.set_from_single_run(input_dir=input_dir, filename=filename,
+                                            hist_mean=hist_mean_dir, bbox=bbox,
+                                            yearrange=yearrange,
+                                            crop=((histmean_prop[2]).split('-'))[0],
+                                            irr=((histmean_prop[2]).split('-'))[1], unit=unit)
+        filename_list.append(('crop_production_' + ((histmean_prop[2]).split('-'))[0] + '-'
+                              + (((histmean_prop[2]).split('-'))[1]).split('.')[0] + '_'
+                              + str(yearrange[0]) + '-' + str(yearrange[1]) + '.hdf5'))
+        output_list.append(crop_production)
+        crop_production.write_hdf5(os.path.join(output_dir, 'Exposure', filename_list[-1]))
+
+    if not return_data:
         return filename_list
     return filename_list, output_list
 
-def normalize_with_fao_cropyield(exp_firr, exp_noirr, input_dir=INPUT_DIR,
-                                 yearrange=np.array([2008, 2018]),
-                                 unit='t', returns='all'):
-    """Normalize the given exposures countrywise with the mean cropyield production quantity
+def normalize_with_fao_cp(exp_firr, exp_noirr, input_dir=INPUT_DIR,
+                          yearrange=np.array([2008, 2018]), unit='t', return_data=True):
+    """Normalize the given exposures countrywise with the mean crop production quantity
     documented by the FAO.
 
-        Parameters:
-        exp_firr (cropyield_isimip): exposure under full irrigation
-        exp_noirr (cropyield_isimip): exposure under no irrigation
+    Parameters:
+        exp_firr (crop_production): exposure under full irrigation
+        exp_noirr (crop_production): exposure under no irrigation
+
+    Optional Parameters:
         input_dir (str): directory containing exposure input data
-        yearrange (array): the mean yield in this year range is used to normalize the exposure
-            data (default 2008-2018)
+        yearrange (array): the mean crop production in this year range is used to normalize
+            the exposure data (default 2008-2018)
         unit (str): unit in which to return exposure (t/y or USD/y)
-        returns (str): returned output
-            'all': country list, ratio = FAO/ISIMIP, normalized exposures, yield per country
-            as documented by the FAO and calculated by the ISIMIP dataset
-            else: country list, ratio = FAO/ISIMIP, normalized exposures
-
+        return_data (str): returned output
+            True: returns country list, ratio = FAO/ISIMIP, normalized exposures, crop production
+            per country as documented by the FAO and calculated by the ISIMIP dataset
```

Comment:
Is there somewhere a link to this FAO? What does FAO stand for?

Reply:
Yes, in the very beginning of the file (line 83-90). We could maybe refer to that here?

Reply:
yes just a short reference would be good guidance here